requirements/scenarios.md

```diff
@@ -253,9 +253,9 @@ A developer accidentally discloses the private key they use to certify their sof
 
 **Implications of this requirement:**
 
-1. All Notary v2 implementations support key revocation as part of their implementation to assure signed content is still valid content.
+1. All Notary Project specification implementations support key revocation as part of their implementation to assure signed content is still valid content.
 1. Registry operators routinely check for revoked keys and remediate the exploited content.
-1. Notary v2 clients routinely check for revoked keys and block the content.
+1. Notary Project specification implementations routinely check for revoked keys and block the content.
```

Comment:
The Notary Project specification implementations routinely

Reply:
I will update it to the `implementations of the Notary Project specification`. Is it better?